### ISSUE TITLE
Schema view doesn't render after a number with default value is defined

### ISSUE THREAD

**Issue by Ryan-McMillan** [tivac/anthracite#102]
Using a number with a default value causes the schema view to not render the next time it's loaded.
```
max : {
    type  : "number",
    attrs : {
        min   : 1,
        max   : 80,
        value : 80
    }
}
```

The error is: input.js:26 Uncaught TypeError: Cannot read property 'child' of undefined
I changed the check on line 25 of input.js to ```val && options.root``` to work around this, no idea if it's the right solution.

**Comment by tivac** [comment]
Yeah that's the correct fix, send a PR?

**Comment by tivac** [comment]
Fixed by 7584c3a7c0e3ac720c133a1f3c4022fff206e89d
